apps/api/src/person/dto/update-person.dto.ts

```diff
@@ -1,14 +1,56 @@
+import { ApiProperty } from "@nestjs/swagger"
+import { Expose } from "class-transformer"
+import { IsString } from "class-validator"
+
 export class UpdatePersonDto {
+  @ApiProperty()
+  @Expose()
+  @IsString()
   firstName?: string
+
+  @ApiProperty()
+  @Expose()
+  @IsString()
   lastName?: string
+
+  @ApiProperty()
+  @Expose()
+  @IsString()
   email?: string
+
+  @ApiProperty()
+  @Expose()
   emailConfirmed?: boolean
```

Comment:
Please remove the `emailConfirmed` field from the update person DTO.

Only the system should be able to confirm their email, people should not be allowed to confirm their email without a real email sent to them